From 9e7fe25aebc4fe4a2c1af15447b44f50c3c97b2f Mon Sep 17 00:00:00 2001
From: Eli Zaretskii <eliz@gnu.org>
Date: Fri, 14 Oct 2016 10:08:25 +0300
Subject: [PATCH 011/122] Include strings.h where available

gdb/ChangeLog

2016-10-14  Eli Zaretskii  <eliz@gnu.org>

	* common/common-defs.h [HAVE_STRINGS_H]: Include strings.h if
	available, to get prototypes of 'strcasecmp' and 'strncasecmp'.

(cherry picked from commit 8ffc1bb12a22e548835c9291871ad0eb68b7f6f0)
---
 gdb/ChangeLog            | 5 +++++
 gdb/common/common-defs.h | 3 +++
 2 files changed, 8 insertions(+)

diff --git a/gdb/ChangeLog b/gdb/ChangeLog
index 47a2b7d6d1..5c53747d4a 100644
--- a/gdb/ChangeLog
+++ b/gdb/ChangeLog
@@ -1,3 +1,8 @@
+2016-10-14  Eli Zaretskii  <eliz@gnu.org>
+
+	* common/common-defs.h [HAVE_STRINGS_H]: Include strings.h if
+	available, to get prototypes of 'strcasecmp' and 'strncasecmp'.
+
 2016-10-12  Yao Qi  <yao.qi@linaro.org>
 
 	PR tdep/20682
diff --git a/gdb/common/common-defs.h b/gdb/common/common-defs.h
index 2c94117045..6719ea675a 100644
--- a/gdb/common/common-defs.h
+++ b/gdb/common/common-defs.h
@@ -49,6 +49,9 @@
 #include <stdint.h>
 
 #include <string.h>
+#ifdef HAVE_STRINGS_H
+#include <strings.h>	/* for strcasecmp and strncasecmp */
+#endif
 #include <errno.h>
 #include <alloca.h>
 
-- 
2.11.0

